examples/app/components/ComponentWithEslintMandatoryProp/example.tsx

```diff
@@ -0,0 +1,19 @@
+import { ComponentWithEslintMandatoryProp } from "examples/app/components/ComponentWithEslintMandatoryProp";
+
+export const Usage = () => {
+  return (
+    <>
+      <ComponentWithEslintMandatoryProp
+        someMandatoryProp
+        eslintMandatoryProp
+        otherEslintMandatoryProp
+      />
+      {/* This component is missing both `eslintMandatoryProp` and `otherEslintMandatoryProp` and should show two eslint errors */}
+      {/* eslint-disable-next-line plentific/enforce-eslint-mandatory-prop */}
+      <ComponentWithEslintMandatoryProp someMandatoryProp />
+      {/* This component is missing `otherEslintMandatoryProp` and should show an eslint error */}
+      {/* eslint-disable-next-line plentific/enforce-eslint-mandatory-prop */}
+      <ComponentWithEslintMandatoryProp someMandatoryProp eslintMandatoryProp />
```

Comment:
Possible improvement here - this rule is based on a component name, so if we would have

```tsx
import ComponentWithEslintMandatoryProp from './path/a'

() => return <ComponentWithEslintMandatoryProp/>
```

```tsx
import ComponentWithEslintMandatoryProp from './path/b'

() => return <ComponentWithEslintMandatoryProp/>
```

both will fail because the name is the same. Up to us if we need to make it more detailed, and actually define a path to the component in the config instead of it's name alone. But actually maybe it will be alright, as we don't have any clashing names in our shared components which will use this eslint rule. But maybe other guilds if will need this to be more unique